content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -12,35 +12,78 @@ weight: 30
+  - Links should primarily be to the official Kubernetes documentation. When using external references, links should be diverse - For example a submission shouldn't contain only links back to a single company's blog.
+  - This can sometime be a delicate balance. The [blog team](https://kubernetes.slack.com/messages/sig-docs-blog/) is there to give guidance on whether a post is appropriate for the Kubernetes blog, so don't hesistate to reach out. 
+- Blog posts do not offer deadlines.
+    - Articles are reviewed by community volunteers and **can not guarantee** any time-based deadlines, though we will try our best to accomodate you.
+  - Many core parts of the project are submitting posts during release windows and publication time will be delayed. Consider submitting during a quieter period of the release cycle.
+  - If you are looking for greater coordination on post release dates, coordinating with [CNCF marketing](https://www.cncf.io/about/contact/) is a more appropriate choice than submitting a blog post.
+  - Sometimes reviews can get backed up. If you feel your review isn't getting the attention it needs, you can reach out to the blog team via [this slack channel](https://kubernetes.slack.com/messages/sig-docs-blog/) to ask in real time. 
+- Blog posts should be relevant to Kubernetes users.
+  - Topics related to participation in or results of Kubernetes SIGs activities are always on topic (see the work in the [Upstream Marketing Team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/blog-guidelines.md#upstream-marketing-blog-guidelines) for support on these posts). 
+  - The components of Kubernetes are purposely modular, so tools that use existing integration points like CNI and CSI are on topic. 
+  - Posts about other CNCF projects may or may not be on topic. We recommend asking the blog team before submitting a draft.
+    - Many CNCF projects have their own blog. These are often a better choice for posts. There are times of major feature or milestone for a CNCF project that users would be interested in reading on the Kubernetes blog.
+- Blog posts should be original content
+    - The official blog is not for repurposing existing content from a third party as new content.
+    - The [license](https://github.com/kubernetes/website/blob/master/LICENSE) for the blog does allow commercial use of the content for commercial purposes, just not the other way around. 
+- Blog posts should aim to be future proof
+  - Given the development velocity of the project, we want evergreen content, that won't require updates to stay accurate to the reader. 
+  - It can be a better choice to add a tutorial or update official documentation, then write a higher level overview as a blog post.
```

Comment:
```suggestion
  - It can be a better choice to add a tutorial or update official documentation than to write a high level overview as a blog post.
```